CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 1.8.0 / 2017-10-06
+
+* [CHANGE] Rule links link to the _Console_ tab rather than the _Graph_ tab to
+  not trigger expensive range queries by default.
+* [CHANGE] Changed help link to `https://prometheus.io/docs`.
```

Comment:
These are both an ENHANCEMENTs as there no stability guarantees for the UI

Reply:
I'd read "no stability guarantee for the UI" as "UI changes will not cause a major release" rather than "changes are rebranded as enhancements".

Reply:
Neither of these are breaking changes, they're minor UI tweaks.

Reply:
One person's bug fix is another persons breaking change. Obligatory reference: https://xkcd.com/1172/ 

Ironically, my original plan was to have the rule link change as a bug fix. I was stopped by a @brian-brazil .

While I cannot imagine that anybody really depends on https://prometheus.io being linked from the help button, I can totally imagine that some users depend on the exact shape and target of links in the emitted alerts. Calling it out as a change is merely conservative.

However, I don't want to spend any more energy in a discussion about a relatively insignificant topic. Please could anybody that is not one of us two make a call?

Reply:
Ah, it also affects the generatorURL. That'd make it a CHANGE alright.

Reply:
OK. I'll move the help link to `[ENHANCEMENT]` and leave the rule link as is. Just a sec…